src/prompts/template.ts

```diff
@@ -57,22 +57,25 @@ export const templateMissingOptionPrompt = async (options, folderNameAnswers, de
       name: 'template',
       message: 'Please choose which node-mongo API boilerplate template to use',
       choices: templateCollection, //uses full names of templates in prompt for better user experience
-      default: defaultTemplate
+      default: defaultTemplate,
+      template: defaultTemplate
```

Comment:
Is there a reason why you added `template: defaultTemplate` here? @tylerjusfly 
> This doesn't look like it was there before from what the file changes says.

Reply:
Help remove it back.

Reply:
yes, it would be nice to talk about it in our next meeting.